src/index.js

```diff
@@ -1,5 +1,20 @@
-export * from './addons'
-export * from './collections'
-export * from './elements'
-export * from './modules'
-export * from './views'
+import { version } from '../package.json'
+// Heads Up!
+//
+// Do not replace this with `export * from '...'` syntax.
+// We need to export an object here for browser builds.
+// Otherwise, we end up with every component on the window.
+import * as addons from './addons'
+import * as collections from './collections'
+import * as elements from './elements'
+import * as modules from './modules'
+import * as views from './views'
+
+export default {
+  ...addons,
+  ...collections,
+  ...elements,
+  ...modules,
+  ...views,
+  version,
```

Comment:
I've added the package version to the exported object.  Available in the browser and in the commonjs build.